spanner/src/delete_dml_returning.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * For instructions on how to run the full sample:
+ *
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/spanner/README.md
+ */
+
+namespace Google\Cloud\Samples\Spanner;
+
+// [START spanner_delete_dml_returning]
+use Google\Cloud\Spanner\SpannerClient;
+
+/**
+ * Delete data from the given database using DML returning feature.
+ *
+ * @param string $instanceId The Spanner instance ID.
+ * @param string $databaseId The Spanner database ID.
+ */
+function delete_dml_returning(string $instanceId, string $databaseId): void
+{
+    $spanner = new SpannerClient();
+    $instance = $spanner->instance($instanceId);
+    $database = $instance->database($databaseId);
+
+    $transaction = $database->transaction();
+
+    // DML returning sql delete query
+    $result = $transaction->execute(
+        'DELETE FROM Singers WHERE FirstName = @firstName '
+        . 'THEN RETURN *',
+        [
+            'parameters' => [
+              'firstName' => 'Melissa',
+            ]
+        ]
+    );
+    foreach ($result->rows() as $row) {
+        printf(
+            'Row (%s, %s, %s) deleted\n' . PHP_EOL,
```

Comment:
```suggestion
            'Row (%s, %s, %s) deleted' . PHP_EOL,
```

Can you remove it from other samples too?

Reply:
Thanks, fixed in [de8e44b](https://github.com/GoogleCloudPlatform/php-docs-samples/pull/1721/commits/de8e44bd199f023ac080a6a21e409e6d98943f58).